extensions/amp-auto-ads/0.1/ad-tracker.js

```diff
@@ -90,6 +138,23 @@ export class AdTracker {
       }
     });
   }
+
+  /**
+   * @private
+   * @return {number}
+   */
+  getMinAdSpacing_() {
+    const adCount = this.getAdCount();
+    if (this.subsequentMinSpacing_.length == 0 ||
```

Comment:
This can be simplified into a min search:

```js
const adCount = this.getAdCount();
let spacing = this.initialMinSpacing_;
for (let i = 0; i < this.subsequentMinSpacing_.length; i++) {
  const item = this.subsequentMinSpacing_[i];
  if (item.adCount <= adCount) {
    spacing = item.spacing;
  }
}

return spacing;